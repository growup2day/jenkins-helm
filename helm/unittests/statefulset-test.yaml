templates:
  - statefulset.yaml
tests:
  - it: default pod spec should match snapshot
    set:
      image:
        tag: v4.7.2
    asserts:
      - matchSnapshot:
          path: spec.template.spec
