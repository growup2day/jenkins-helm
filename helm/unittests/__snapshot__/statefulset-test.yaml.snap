default pod spec should match snapshot:
  1: |
    containers:
    - env:
      - name: JENKINS_JAVA_OVERRIDES
        value: -Dcasc.reload.token=cacb1c9dc181fd1ca14b64b82b9cf3a16f9577e1623a9a32b6b14da2418cf871
      - name: CASC_JENKINS_CONFIG
        value: /var/lib/jenkins-jcasc
      - name: CONTAINER_HEAP_PERCENT
        value: "0.9"
      - name: KUBERNETES_TRUST_CERTIFICATES
        value: "true"
      - name: OPENSHIFT_ENABLE_OAUTH
        value: "true"
      - name: OPENSHIFT_ENABLE_REDIRECT_PROMPT
        value: "true"
      - name: OVERRIDE_PV_PLUGINS_WITH_IMAGE_PLUGINS
        value: "true"
      image: image-registry.openshift-image-registry.svc:5000/digital-image-builds/jenkins-helm:v4.7.2
      imagePullPolicy: IfNotPresent
      livenessProbe:
        failureThreshold: 30
        httpGet:
          path: /login
          port: 8080
        initialDelaySeconds: 420
        timeoutSeconds: 30
      name: jenkins
      readinessProbe:
        httpGet:
          path: /login
          port: 8080
        initialDelaySeconds: 10
        timeoutSeconds: 30
      resources:
        limits:
          cpu: "2"
          memory: 4Gi
        requests:
          cpu: 200m
          memory: 500Mi
      terminationMessagePath: /dev/termination-log
      volumeMounts:
      - mountPath: /var/lib/jenkins
        name: RELEASE-NAME-jenkins-data
      - mountPath: /var/lib/jenkins-jcasc
        name: jcasc
        readOnly: true
      - mountPath: /var/lib/jenkins-utils
        name: utils
        readOnly: true
    serviceAccountName: RELEASE-NAME-jenkins
    volumes:
    - name: RELEASE-NAME-jenkins-data
      persistentVolumeClaim:
        claimName: RELEASE-NAME-jenkins
    - configMap:
        name: RELEASE-NAME-jenkins-jcasc
      name: jcasc
    - configMap:
        name: RELEASE-NAME-jenkins-utils
      name: utils
